2023-03_sprint/sprint_parties/community_partners.md

```diff
@@ -0,0 +1,73 @@
+# Community Partnership
+
+(become_community_partner)=
+## Become a Community Partner
+
+Community Partners are organizations who support our mission and share our event for outreach.  Data Umbrella will add the Name, Logo and Website Link to this page to bring visibility to your community.  If you would like to be a community partner, please contact us ({{ du_email }}).
+
+We are providing a social media kit with information for sharing with your community:
+
+::::{div} sd-d-flex-row sd-align-major-center
+:::{button-ref} media_kit_ref
+:color: primary
+:ref-type: myst
+:class: sd-fs-5
+
+Social Media Kit {material-regular}`share;3em`
+:::
+::::
+
+## Community Partners
+
+::::{grid} 1 1 2 2
+:::{grid-item-card}
+:link: https://nyc.pyladies.com
+:shadow: none
+:class-card: border-0
+
+![New York City PyLadies](../../_static/community_partners/nyc_pyladies.jpg)
+
+We are the New York City chapter of PyLadies, an international community of women and non-binary people in tech.
+:::
+
+<!-- :::{grid-item-card}
+:link: https://www.womeninanalytics.com/
+:shadow: none
+:class-card: border-0
+
+![Women in Analytics](../../_static/community_partners/wia.png)
+
+An Analytics Community. Featuring Women. For Everyone.
+::: -->
+
+<!-- :::{grid-item-card}
+:link: https://twitter.com/pyladies_co
+:shadow: none
+:class-card: border-0
+
+![PyLadies Colombia](../../_static/community_partners/pyladies_colombia.png)
+
+We are a community that motivates women to use technology and code. Everyone is welcome here, and it is a diverse and inclusive group.
+::: -->
+
+<!-- :::{grid-item-card}
+:link: https://wbds.la
+:shadow: none
+:class-card: border-0
+
+![Women in Bioinformatics & Data Science LA (Latin America)](../../_static/community_partners/wbds_latin_america.png)
+
+Fostering collaboration among women.
+::: -->
+
+
+<!-- :::{grid-item-card}
+:link: https://www.pythonchile.cl
+:shadow: none
+:class-card: border-0
+
+![Python Chile](../../_static/community_partners/pythonchile.png)
+
+Official Chilean Python Community, a place to meet, learn, and empower Chilean Python developers.
+::: -->
```

Comment:
```suggestion
```

I would delete this, otherwise it is difficult to see the starts and closings of the fences. They will still be in the last year source, so we can easily copy and paste them if they are community partners again.